game/views.py

```
@@ -33,6 +33,17 @@
 logger = logging.getLogger(__name__)
 from game.services import cleanup_stale_games
 
+
+def _rate_limit(request, key_prefix, limit=5, window=60):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_rate_limit`

Add return type annotation: `bool`

(ANN202)